examples/http_server/module/src/lib.rs

```diff
@@ -266,7 +266,8 @@ impl CommandHandler for RedirectHelper {
     type Command = RedirectInvocation;
 
     fn handle_command(&mut self, invocation: RedirectInvocation) -> anyhow::Result<()> {
-        let request = invocation.request_receiver.unwrap().receive()?;
+        let request = invocation.request_receiver.as_ref().unwrap().receive()?;
```

Comment:
Is this `as_ref` needed? BTW I'm just asking because I don't uderstand what it does, (I don't remember using `as_ref` on anything other than strings) could you give me a summary?

Reply:
This one is needed, to borrow `invocation.request_receiver` instead of moving it. It was ok to move previously, because we did not close channel before.